data/storage/mysql/create_db_with_users.sql

```diff
@@ -7,3 +7,9 @@ DROP DATABASE IF EXISTS cgrates;
 CREATE DATABASE cgrates;
 
 GRANT ALL on cgrates.* TO 'cgrates'@'localhost' IDENTIFIED BY 'CGRateS.org';
+
+-- extra DB for ers
+DROP DATABASE IF EXISTS cgrates2;
```

Comment:
Please create a separate script if this is needed for some testing since this one is part of the install of the users.